.github/workflows/build.yml

```diff
@@ -11,18 +11,16 @@ jobs:
     strategy:
       matrix:
         channel: 
-          - beta
           - stable
-          - dev
```

Comment:
This are causing problem in the Ci with dependencies resolution, so for now I remove it